lfc/core/src/main/kotlin/org/lflang/generator/uc/UcCmakeGenerator.kt

```diff
@@ -52,7 +52,8 @@ abstract class UcCmakeGenerator(
             |set(LF_MAIN_TARGET ${mainTarget})
             |set(CMAKE_BUILD_TYPE ${targetConfig.getOrDefault(BuildTypeProperty.INSTANCE)})
             |set(PLATFORM POSIX CACHE STRING "Target platform")
-            |include($S{CMAKE_CURRENT_SOURCE_DIR}/Include.cmake)
+            |#include($S{CMAKE_CURRENT_SOURCE_DIR}/src-gen/$S{LF_MAIN}/$S{FEDERATE}/Include.cmake)
```

Comment:
We shouldnt code-generated commented out code. And this code change includes the same file but makes it more brittle by depending on a relative path